docs/content/latest/faq/yugabyte-platform.md

```diff
@@ -14,92 +14,85 @@ isTocNested: false
 
 {{< /note >}}
 
-**Permissions necessary for an Internet-connected host**
-
-- Connectivity to the Internet, either directly or via an HTTP proxy
-- Ability to install and configure [docker-engine](https://docs.docker.com/engine/)
-- Ability to install and configure [Replicated](https://www.replicated.com/), which is a containerized application itself and needs to pull containers from its own Replicated.com container registry
-- Ability to pull Yugabyte container images from [Quay.io](https://quay.io/) container registry, this will be done by Replicated automatically
-
-**Permissions necessary for an airgapped host**
-
-An _airgapped_ host has no path to inbound or outbound Internet traffic at all. For such hosts, the installation is performed as a `sudo` user.
-
-**Additional requirements**
+Yugabyte Platform also requires the following:
 
-For airgapped hosts, a supported version of docker-engine (currently 1.7.1 to 17.03.1-ce). If you do not have docker-engine installed, follow the instructions [here](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/) to first install docker-engine.
+- Connectivity to the Internet, either directly or via an HTTP proxy.
+- Ability to install and configure [docker-engine](https://docs.docker.com/engine/).
+- Ability to install and configure [Replicated](https://www.replicated.com/), which is a containerized application itself and needs to pull containers from its own Replicated.com container registry.
+- Ability to pull Yugabyte container images from [Quay.io](https://quay.io/) container registry (this will be done by Replicated automatically).
+- The following ports open on the platform host: `8800` (replicated ui), `80` (http access to the Yugabyte Platform Console), `22` (ssh).
+- Attached disk storage (such as persistent EBS volumes on AWS): 100 GB SSD minimum.
+- A Yugabyte Platform license file from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form).
+- Ability to connect from the Yugabyte Platform host to all YugabyteDB data nodes via SSH.
 
-- Following ports should be open on the YugaWare host: `8800` (replicated ui), `80` (http for YugabyteDB Admin Console), `22` (ssh)
-- Attached disk storage (such as persistent EBS volumes on AWS): 100 GB minimum
-- A Yugabyte Platform license file from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form)
-- Ability to connect from the Yugabyte Platform host to all YugabyteDB data nodes. If this is not set up, [setup passwordless ssh](#step-5-troubleshoot-yugaware).
+For a complete list of prerequisites, refer to [Prerequisites](../../yugabyte-platform/install-yugabyte-platform/prerequisites/).
 
-## What are the OS requirements and permissions to run the YugabyteDB data nodes?
+## What are the OS requirements and permissions to run YugabyteDB data nodes?
```

Comment:
```suggestion
## What are the OS requirements to run YugabyteDB data nodes?
```
There seems to be no information about _permissions_ under the Deployment checklist